cluster/gce/container-linux/configure-helper.sh

```diff
@@ -360,6 +363,30 @@ current-context: service-account-context
 EOF
 }
 
+function create-kubescheduler-kubeconfig {
```

Comment:
no. presumably most of those are still running the scheduler against the unsecured port as well.